gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/work/GenerateWorkUnitsResult.java

```diff
@@ -27,19 +26,14 @@
 
 
 /**
- * Data structure representing the stats for a committed dataset, and the total number of committed workunits in the Gobblin Temporal job
- * Return type of {@link org.apache.gobblin.temporal.ddm.workflow.ProcessWorkUnitsWorkflow#process(WUProcessingSpec)}
- * and {@link org.apache.gobblin.temporal.ddm.workflow.CommitStepWorkflow#commit(WUProcessingSpec)}.
+ * Data structure representing the result of generating work units, where it returns the number of generated work units and
+ * the folders that the parent workflow should clean up as a side effect of generating WUs
```

Comment:
nit: "as a side effect of" seems less accurate than "as a follow up to"

Reply:
Ah I missed this, yes I modified the javadoc now